components/brave_wallet_ui/common/async/hardware.ts

```diff
@@ -20,14 +20,14 @@ import {
   HardwareVendor
 } from '../api/hardware_keyrings'
 import { TrezorErrorsCodes } from '../hardware/trezor/trezor-messages'
-import FilecoinLedgerKeyring from '../hardware/ledgerjs/filecoin_ledger_keyring'
 import TrezorBridgeKeyring from '../hardware/trezor/trezor_bridge_keyring'
 import EthereumLedgerBridgeKeyring from '../hardware/ledgerjs/eth_ledger_bridge_keyring'
 import SolanaLedgerBridgeKeyring from '../hardware/ledgerjs/sol_ledger_bridge_keyring'
 import { BraveWallet } from '../../constants/types'
 import { LedgerEthereumKeyring, LedgerFilecoinKeyring, LedgerSolanaKeyring } from '../hardware/interfaces'
 import { EthereumSignedTx } from '../hardware/trezor/trezor-connect-types'
 import { SignedLotusMessage } from '@glif/filecoin-message'
```

Comment:
We should remove all use of this library that is outside the untrusted frame, otherwise we haven't really isolated it and don't get the security benefits. You may need to manually define types like SignedLotusMessage in fil-ledger-messages.ts instead of importing. We did this with the trezor-connect library https://github.com/brave/brave-core/pull/14564.

I believe there's also usage in components/brave_wallet_ui/common/hardware/types.ts.